content/docs/deploying/k8s/quickstart.mdx

```diff
@@ -72,14 +72,14 @@ job.batch/pomerium-gen-secrets created
 ingressclass.networking.k8s.io/pomerium created
 ```
 
-1. Add the certificate [created earlier](#certificates) and key to the cluster as a Secret:
+2. Add the certificate [created earlier](#certificates) and key to the cluster as a Secret:
```

Comment:
Heads up -- I think using `1.` repeated is valid markdown and is easier to maintain long term. 

https://www.markdownguide.org/basic-syntax/#lists-1

Reply:
I'm not sure why, but the list items aren't incrementing on the guide. So, each '1.' starts a new ordered list. Here's a pic after inspecting the page:
<img width="927" alt="SCR-20221206-mmv" src="https://user-images.githubusercontent.com/60552605/206025153-fcbe053a-7c63-4496-b450-d3510ba32364.png">

Reply:
it _could_ be the indentation? In any case, either is valid as far as I am concerned. They should be syntactically identical / interpreted by markdown the same way.